### DISCUSSION TITLE
Consider collaboration with @gitattributes org

### DISCUSSION THREAD

**Discussion by jcbhmr** [spenserblack/gh-gitattributes#4] [Ideas]
There's the cool @gitattributes org which would be really cool 😎 to put this repo under to make it very pseudo-official. It undeniably has a better branding/name 😜

**Comment by spenserblack** [comment]
Yeah, I'm not opposed if they'll have me :slightly_smiling_face:

**Comment by jcbhmr** [comment]
Heyyy @jozefizso 👋 would putting this https://github.com/spenserblack/gh-gitattributes project under the @gitattributes organization be cool with you? (asking on the behalf of @spenserblack)

<sup>P.s. if you don't already, github recommends two organization owners so it might be worth adding spenserblack or one of the other big-time https://github.com/alexkaratarakis/gitattributes people in case you ever go on vacation or loose interest or something 🤷‍♂️
https://docs.github.com/en/organizations/collaborating-with-groups-in-organizations/best-practices-for-organizations

**Comment by jozefizso** [comment]
Sure, I think we can host this repo under @gitattributes org.

**Reply by spenserblack** [reply]
Cool. I'm thinking that `gitattributes/gh-gitattributes` can be a hard fork of this repo. Then, depending on the level of access you want to give me, I could be a collaborator, team member, or just a frequent contributor :slightly_smiling_face:

Then, the to-do AFAIK is
- [ ] Change the default source repository
- [ ] Make the UI more similar to the web app -- cumulative, where multiple gitattribute settings can be selected
